packages/design-system/src/components/contextMenu/styled.js

```diff
@@ -52,38 +48,7 @@ export const Shadow = styled.div`
 `;
 
 export function SmartPopover(props) {
```

Comment:
Why are props not deconstructed here like we always do? We could just do `{ isOpen, ...rest }` - does the `Popover` ever need the `isOpen` property?

Reply:
I just saw that `Popover` is exported and used somewhere else (with `isOpen`) so I didn't want to change it, thus we cannot do this (because `isOpen` is needed):
![image](https://user-images.githubusercontent.com/21036224/172597388-efec8844-116c-4bf9-82ce-2d0615c19c34.png)

Reply:
But `isOpen` will always be true when we use this `SmartPopover`, so it would just be:

```js
function SmartPopover({ isOpen, ...rest }) {
  return isOpen ? <Popover isOpen {...rest} /> : null;
}
```

Alternatively, you could set `isOpen` to default `true` in `Popover` (it already has a `.defaultProps` config).

Reply:
Sure, it's just a detail at this point (and 101/94 chars vs 87 like we have right now 😉 ).

Reply:
I just don't like having this one component not use prop destructuring like we do everywhere else. It's important to me, okay? 😄